packages/http-client-java/generator/http-client-generator-test/src/main/java/com/cadl/protocolandconvenient/models/ResourceD.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.cadl.protocolandconvenient.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * The ResourceD model.
+ */
+@Immutable
+public final class ResourceD implements JsonSerializable<ResourceD> {
```

Comment:
TCGC returns the `usage = 256`, not `0`. But it is only used by protocol method. Would need @tadelesh's input. 
test case: https://github.com/microsoft/typespec/blob/main/packages/http-client-java/generator/http-client-generator-test/tsp/protocol-api-convenient-api.tsp#L131-L136